extensions/interactions/CodeRepl/CodeRepl.js

```diff
@@ -119,7 +119,7 @@ oppia.directive('oppiaInteractiveCodeRepl', [
           execLimit: 10000
         });
 
-        $scope.runCode = function(codeInput) {
+        $scope.runAndSubmitCode = function(codeInput, status) {
```

Comment:
Better to remove the extra arg and do it as suggested before, I think. Sorry about the old test confusing things!